python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -446,7 +450,7 @@ def makeMinimalSchema(filterNameList, *, addCentroid=False,
                 prefix="coord",
                 names=["ra", "dec"],
                 units=["rad", "rad"],
-                diagonalOnly=(coordErrDim == 2),
+                diagonalOnly=((coordErrDim == 2) or (fullCoordinateCovariance)),
```

Comment:
Doesn't this produce `coord_ra_dec_Cov`, not `coord_ra_coord_dec_Cov` as advertised? In fact, it seems that if `coordErrDim > 0` and `fullCovarianceMatrix`, then you get *both* columns.

Unfortunately, I don't see an easy fix short of modifying `addFields` to support empty prefixes; the code will inject a leading underscore if you just pass `prefix=""`.

Reply:
It's correct that this produces `coord_ra_dec_Cov`, not `coord_ra_coord_dec_Cov` (which is the way `fullCoordinateCovariance` names the field). I think the only solution would be to just have a separate `addFields` command to add the covariance. We can discuss during the meeting today.